kroxylicious-runtime/src/test/java/io/kroxylicious/proxy/filter/DenyCipherSuiteFilterTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.proxy.filter;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+public class DenyCipherSuiteFilterTest {
+
+    @Test
+    public void emptyCiphersDefaultsToDefaultCiphers() {
+        List<String> defaultCiphers = Arrays.asList("CIPHER_SUITE_1", "CIPHER_SUITE_2", "CIPHER_SUITE_3", "CIPHER_SUITE_4");
+        Set<String> supportedCiphers = new HashSet<>(Arrays.asList("CIPHER_SUITE_1", "CIPHER_SUITE_2", "CIPHER_SUITE_3"));
+        List<String> defaultAllowedCiphers = Arrays.asList("CIPHER_SUITE_1", "CIPHER_SUITE_2", "CIPHER_SUITE_3");
+
+        String[] filtered = DenyCipherSuiteFilter.INSTANCE.deniedCiphers(null)
+                .filterCipherSuites(null, defaultCiphers, supportedCiphers);
+        assertArrayEquals(defaultAllowedCiphers.toArray(), filtered);
```

Comment:
we are favouring assertj for all new word.. it is just much more expressive and concise.

```suggestion
        assertThat(filtered).containsAll(defaultAllowedCiphers);
```